sdk/communication/azure-communication-rooms/azure/communication/rooms/_rooms_client.py

```diff
@@ -0,0 +1,300 @@
+        :param valid_until: (Optional)The timestamp from when the room can no longer be joined. The timestamp
+         is in RFC3339 format: ``yyyy-MM-ddTHH:mm:ssZ``.
+        :type valid_until: (Optional)datetime
+        :param room_join_policy: (Optional)The join policy of the room.
+        :type room_join_policy: (Optional)RoomJoinPolicy
+        :param participants: (Optional) Collection of identities invited to the room.
+        :type participants: (Optional)List[RoomParticipant]
+        :returns: Created room.
+        :rtype: ~azure.communication.rooms.CommunicationRoom
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        create_room_request = CreateRoomRequest(
+            valid_from=valid_from,
+            valid_until=valid_until,
+            room_join_policy=room_join_policy,
+            participants=[p.to_room_participant_internal() for p in participants] if participants else None
+        )
+
+        repeatability_request_id = uuid.uuid1()
+        repeatability_first_sent = datetime.utcnow()
+
+        create_room_response = self._rooms_service_client.rooms.create_room(
+            create_room_request=create_room_request,
+            repeatability_request_id=repeatability_request_id,
+            repeatability_first_sent=repeatability_first_sent,
+            **kwargs)
+        return CommunicationRoom.from_room_response(create_room_response)
+
+    @distributed_trace
+    def delete_room(
+        self,
+        room_id: str,
+        **kwargs
+    ) -> None:
+
+        """Delete room.
+
+        :param room_id: Required. Id of room to be deleted
+        :type room_id: str
+        :returns: None.
+        :rtype: None
+        :raises: ~azure.core.exceptions.HttpResponseError
+
+        """
+        self._rooms_service_client.rooms.delete_room(room_id=room_id, **kwargs)
+
+    @distributed_trace
+    def update_room(
+        self,
+        *,
+        room_id: str,
+        valid_from: Optional[datetime] = None,
+        valid_until: Optional[datetime] = None,
+        room_join_policy: Optional[RoomJoinPolicy] = None,
+        participants: Optional[List[RoomParticipant]] = None,
+        **kwargs
+    ) -> CommunicationRoom:
+        """Update a valid room's attributes
+
+        :param room_id: Required. Id of room to be updated
+        :type room_id: str
+        :param valid_from: The timestamp from when the room is open for joining. The timestamp is in
+         RFC3339 format: ``yyyy-MM-ddTHH:mm:ssZ``.
+        :type valid_from: (Optional)datetime
+        :param valid_until: The timestamp from when the room can no longer be joined. The timestamp
+         is in RFC3339 format: ``yyyy-MM-ddTHH:mm:ssZ``.
+        :type valid_until: (Optional)datetime
+        :param room_join_policy: (Optional)The join policy of the room.
+        :type room_join_policy: (Optional)RoomJoinPolicy
+        :param participants: (Optional) Collection of identities invited to the room.
```

Comment:
Does this add to or replace the existing participants in the room? That kind of information should be added to the docstring.